accounts/models.py

```diff
@@ -45,6 +52,37 @@ class CustomUser(AbstractUser):
         default=Organisation.DEFAULT_PK
     )
 
+    about = models.TextField(
```

Comment:
Do we really need this "About" field (short description about yourself) on the app?

Reply:
I'm fine with keeping it, if you think it's necessary, but I don't see it's relevance for the actual Hackathon app itself.

Reply:
Again, it's for the profile, not the hackathon itself, but if the teams share the hackathon project there is a link to the profile of each of the students and this will give them the opportunity to write a blurb about themselves.